releases/23.0.0.6/full/Dockerfile.ubi.openjdk11

```diff
@@ -1,15 +1,40 @@
-FROM icr.io/appcafe/ibm-semeru-runtimes:open-11-jdk-ubi
+FROM icr.io/appcafe/ibm-semeru-runtimes:open-11-jdk-ubi AS getRuntime
 
 USER root
 
-ARG LIBERTY_VERSION=23.0.0.5
-ARG LIBERTY_SHA=4ee902cc2b5a775349510c9025af50d90f1c072a
-ARG LIBERTY_BUILD_LABEL=cl230520230514-1901
+ARG LIBERTY_VERSION=23.0.0.6
+ARG LIBERTY_SHA=bd69a21ba94f5707aff2c51174c95af66cf27b37
 ARG LIBERTY_DOWNLOAD_URL=https://repo1.maven.org/maven2/io/openliberty/openliberty-runtime/$LIBERTY_VERSION/openliberty-runtime-$LIBERTY_VERSION.zip
+
+ARG VERBOSE=false
+
+# If there is a local copy of the image use that instead
+COPY resources/ /tmp/
+
+# Install Open Liberty
+RUN yum -y install shadow-utils wget unzip openssl \
+    && if [ ! -f /tmp/wlp.zip ]; then wget -q $LIBERTY_DOWNLOAD_URL -U UA-Open-Liberty-Docker -O /tmp/wlp.zip; fi \
+    && echo "$LIBERTY_SHA  /tmp/wlp.zip" > /tmp/wlp.zip.sha1 \
+    && sha1sum -c /tmp/wlp.zip.sha1 \
+    && chmod -R u+x /usr/bin \
```

Comment:
I believe this should be run in the final stage (as well)

Reply:
See issue #382 can we explain why this would be needed?  Changing attributes on every file in /usr/bin causes our image size to increase.